File-Formats/PDF/Convert-HTML-To-PDF/WPF.md

```diff
@@ -10,15 +10,15 @@ keywords: Assemblies
 # Convert HTML to PDF file in WPF
 ## Steps to convert Html to PDF document in WPF:
 
-Create a new WPF application project.
+1. Create a new WPF application project.
 ![Convert HTMLToPDF WPF Step1](htmlconversion_images/WPF_Step1.png)
 Now, the project configuration window appears. Click Create button to create a new project with the default project configuration.
```

Comment:
Modify - In project configuration window, set the project name, location and .NET Framework version and click _Create_ button to create a new project.

Reply:
Modified the line